src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -474,7 +476,7 @@ export default class FilingHistoryList extends Mixins(
       const item: HistoryItemIF = {
         availableOnPaperOnly: filing.availableOnPaperOnly,
         commentsCount: filing.commentsCount,
-        displayName: filing.displayName,
+        displayName: this.isTypeDissolution(filing) ? this.getFilingHistoryListResource?.title : filing.displayName,
```

Comment:
Don't do this. The API is supposed to provide the display name in all cases. Change the API instead.

Reply:
Rationale: Special cases are the killer of maintainable applications. The ledger design is for the API to say what to display here.

Reply:
https://github.com/bcgov/lear/blob/859dd918f06dbdb11b81622a2ef72a90b3e1011e/legal-api/src/legal_api/core/meta/filing.py#L167 I think?

Reply:
Yes I think that's it.

I'm wondering if it would make sense to return a different code for SP and GP as well? (I'm assuming this is the dissolution type but I could be wrong.) For future conversion of type to name (in enum mixin).

@argush3 What do you think?

Reply:
PS Travis, the API also determines the name of the outputs (PDFs), which probably need the same change.

Reply:
Yes, we should update the API for this.

Reply:
I mean different reason text in the Entity Info header depending on the type of dissolution.

See src/mixins/enum-mixin.ts in this PR and [here where it's used](https://github.com/bcgov/business-filings-ui/blob/0c6eb4b3fc29f46a1fc7a6091e403070bf9af9ff/src/App.vue#L581).

Reply:
Code was reverted.